docs/api/process.md

```diff
@@ -171,6 +171,10 @@ Returns `Object`:
 Returns an object giving memory usage statistics about the entire system. Note
 that all statistics are reported in Kilobytes.
 
+### `process.takeHeapSnapshot()`
+
+Returns `String` - The file path if the heap snapshot has been created successfully.
```

Comment:
The method has to be synchronous, right?

What is returned if there is an error?
If a method throws an exception, it has to be documented.